biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -417,13 +417,27 @@ public Archive binaryArchive() {
 
 	@Override
 	public Map<Program, Dependency> getDependencies(MavenScope scope, boolean transitive) throws Exception {
-		return getDependencies(EnumSet.of(scope), transitive);
+		return getDependencies(EnumSet.of(scope), transitive, false);
+	}
+
+	@Override
+	public Map<Program, Dependency> getDependencies(MavenScope scope, boolean transitive,
+		boolean useDependencyManagement) throws Exception {
```

Comment:
Name this `dependencyManagement` to match other places.

Reply:
This one I would keep, becuse we already have a field that represents the `dependencyManagement` of the pom.

Reply:
But we don't have `useTransitive'. I think the name `dependencyManagement` is fine since it is a boolean and a true or false value makes sense like for `transitive`.